openapi/paths/API@living@application.yaml

```diff
@@ -7,7 +7,8 @@ get:
 
     - can order on `id`,`creationDate`, `createdBy`, `profileId`, `token`, `displayName`, `updatedBy`, `lastUpdateDate`, `version`
     - can search on `token`, `displayName`, `version`
-    - can filter on `token`, `displayName`, `version`, `profileId`, `creationDate`, `createdBy`, `updatedBy` , `lastUpdateDate`, `userId`
+    - can filter on `token`, `displayName`, `version`, `profileId`, `creationDate`, `createdBy`, `updatedBy` , `lastUpdateDate`
```

Comment:
@DumitruCorini What is the difference between
can search on
and 
can filter on
?